app/Presenters/ComponentGroupPresenter.php

```diff
@@ -51,11 +53,25 @@ public function lowest_human_status()
      */
     public function lowest_status_color()
     {
-        if ($component = $this->wrappedObject->enabled_components_lowest()->first()) {
+        if ($component = $this->enabled_components_lowest()) {
             return AutoPresenter::decorate($component)->status_color;
         }
     }
 
+    /**
+     * Return the enabled components from the wrapped object, and cache it if need be.
+     *
+     * @return bool
+     */
+    public function enabled_components_lowest()
+    {
+        if (is_bool($this->enabledComponentsLowest)) {
```

Comment:
Why are we checking `is_bool` if the variable is declared as `false` by default?